frontend/server/src/generated/apis/auth/git_push.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/sh
```

Comment:
Out of curiosity, why do we need such a script here?

Reply:
it's auto generated, I don't mind keeping it here. any concerns?

Reply:
No concern. I was just curious :)